cryptography/src/bls12381/primitives/ops.rs

```diff
@@ -26,57 +76,66 @@ pub fn keypair<R: RngCore>(rng: &mut R) -> (group::Private, group::Public) {
 ///
 /// Signatures produced by this function are deterministic and are safe
 /// to use in a consensus-critical context.
-pub fn sign(private: &group::Private, namespace: &[u8], message: &[u8]) -> group::Signature {
+pub fn sign_message(
```

Comment:
I opted to migrate `sign/verify` to `sign_message`/`verify_message` because they use a different `dst` than Proof-of-Possession stuff.

Reply:
Consider doing `_msg` and `_pop`?